src/main/java/where2eat.ump

```diff
@@ -0,0 +1,68 @@
+namespace ca.mcgill.ecse.where2eat.model;
+
+enum Category {Chinese, Indian, Portuguese}
+
+class Where2Eat {
+  1 <@>- * User users;
+  1 <@>- * Login loginIDs;
+  1 <@>- * Group groups;
+  1 <@>- * Restaurant restaurants;
+  1 <@>- * Location locations;
+  1 <@>- * Address addresses;
+  1 <@>- * PhoneNumber phoneNumbers;
+}
+
+class User
+{
+  String firstName;
+  String lastName;
+  String userID;
```

Comment:
Shouldn't this be an integer? I'm assuming the separation between User and Login is to separate the serialization of meta-data about the User and their credentials?